cartocrow/isoline_simplification/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+set(SOURCES
+	voronoi_helpers.cpp
+	isoline_simplifier.cpp
+	collapse.cpp
+	ipe_bezier_wrapper.cpp
+	symmetric_difference.cpp
+	simple_isoline_painting.cpp
+	ipe_isolines.cpp
+	simple_smoothing.cpp
+)
+set(HEADERS
+	isoline.h
+    types.h
+    voronoi_helpers.h
+	isoline_simplifier.h
+	ipe_bezier_wrapper.h
+	collapse.h
+	ipe_isolines.h
+	simple_isoline_painting.h
+	symmetric_difference.h
+	simple_smoothing.h
+)
+
+add_library(isoline_simplification ${SOURCES})
+target_link_libraries(isoline_simplification
+	PUBLIC core
+	PRIVATE glog::glog
```

Comment:
As far as I can see, the isoline simplification code doesn't use glog anywhere, so it should not be necessary to link to it.

Reply:
Fixed